CONTRIBUTING.md

```diff
@@ -115,7 +115,9 @@ Some examples of valid locale tags:
  - Chinese (Singapore): `zh_SG`.
  - Portuguese (Brazil): `pt_BR`.
 
-A list of translated templates for alias pages can be found in [here](contributing-guides/translation-templates/alias-pages.md).
+A list of translated templates for alias pages can be found [here](contributing-guides/translation-templates/alias-pages.md).
+
+Specific rules of Chinese and French can be found [here](contributing-guides/language-specific-rules.md).
```

Comment:
```suggestion
Language specific rules for pages can be found [here](contributing-guides/language-specific-rules.md).
```
I think having this generalised would be good. As many more languages will be added to this list. For example, @reinhart1010 is working on rewriting all Indonesian pages in #8576.